CONTRIBUTING.md

```diff
@@ -0,0 +1,102 @@
+# Contributing to Tarantool
+
+You likely come here from issues or pull requests page. There are some
+suggestions how to make your contribution most useful.
+
+## File an issue
+
+Consider using bug report / feature request template. Feel free to adopt it for
+your needs. You can opt out and start from a blank issue, but take care to
```

Comment:
1. 'opt-out' is a noun, 'opt out' is a verb. We need a verb here.

    https://www.oxfordlearnersdictionaries.com/definition/english/opt-out_1?q=opt+out

    https://www.oxfordlearnersdictionaries.com/definition/english/opt-out_2?q=opt-out

2. 'care of' -- it is correct. However @patiencedaur suggested me to rewrite the phrase with 'make sure', because

    > <...> take care of the completeness of the information.

    sounds artificial for English. Less heavy varint would be the following.

    > <...> make sure that the information is sufficiently complete.

    I'm not sure that it is better represents what I want to say. In Russian it would be:

    > Можете не пользоваться шаблоном и описать проблему с чистого листа, но не забывайте о полноте информации.

    I don't want to bloat this part of the document with something like 'information, which is necessary to fully understand the request, reproduce the problem or make a decision about improtance of the flaw considering you user scenario'. I want just a small allude that we expect carefully

Reply:
Sorry, misclick.

Reply:
1. 'opt-out' is a noun, 'opt out' is a verb. We need a verb here.

    https://www.oxfordlearnersdictionaries.com/definition/english/opt-out_1?q=opt+out

    https://www.oxfordlearnersdictionaries.com/definition/english/opt-out_2?q=opt-out

2. 'care of' -- it is correct. However @patiencedaur suggested me to rewrite the phrase with 'make sure', because

    > <...> take care of the completeness of the information.

    sounds artificial for English. Less heavy variant would be the following.

    > <...> make sure that the information is sufficiently complete.

    I'm not sure that it is better represents what I want to say. I want to say 'take care of it', not 'ensure it is so'.

    I spin here for a while, but I didn't end up with a variant that would fully satisfy me. I left the original phrase with fixed grammar and 'be mindful of' instead of 'take care of':

    > but be mindful of the completeness of the information.